### PR TITLE
Add post to cluster

### PR DESCRIPTION
**Pull request by hesamemadi**

### CODE REVIEW

File: src/main/java/com/innohub/innosorter/management/IssueManager.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import com.innohub.innosorter.entity.Administrator;
+import com.innohub.innosorter.entity.Cluster;
+import com.innohub.innosorter.entity.Post;
+import com.innohub.innosorter.entity.User;
+
+public class IssueManager {
+
+    public void addPostToCluser(User user, Cluster clusterOne, Post postOne) {
+        if (!(user instanceof Administrator)){
+            throw new RuntimeException("User does not have enough priviledge to do this action.");
```
Comment (aman212121) [inline src/main/java/com/innohub/innosorter/management/IssueManager.java]: Use a string variable instead

**Review by aman212121 — CHANGES_REQUESTED** [review]
Store the exception message in a string variable

**Review by karthiksriramuoa — APPROVED** [review]
Approved

**Review by aman212121 — APPROVED** [review]

**Review by karthiksriramuoa — APPROVED** [review]
Approved

**Pull request merged**